src/Scorecard.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+class Scorecard {
+
+  constructor(){
+    this.frames = 1
+    this.frame_score = []
+    this.scores = []
+    this.running_score = []
```

Comment:
It's unclear to me the differences between each of these arrays of scores. The things that are confusing me:
- we push the roll to both `this.frame_score` and `this.scores` so why do we need both?
- does `this.running_score` need to be an array which is an instance variable? Maybe it's the naming that's confusing me, as a running score to me sounds like it should be a number